modules/javafx.graphics/src/main/java/com/sun/glass/ui/Window.java

```diff
@@ -370,6 +371,10 @@ public void setView(final View view) {
         if (view != null && _setView(this.ptr, view)) {
             this.view = view;
             this.view.setWindow(this);
+            // JDK-8299968: View size update (especially notifyResize event) have to happen
```

Comment:
Minor: we generally don't put a bug ID in comments in the implementation when fixing a bug.